src/widgetastic_patternfly4/select.py

```diff
@@ -134,5 +66,68 @@ def read(self):
         """Returns a string of the text of the selected option."""
         return self.browser.text(self.BUTTON_LOCATOR)
 
-    def __repr__(self):
-        return "{}({!r})".format(type(self).__name__, self.locator)
+
+class CheckboxSelect(Select):
+    ITEMS_LOCATOR = ".//label[contains(@class, 'pf-c-select__menu-item')]"
+    ITEM_LOCATOR = f"{ITEMS_LOCATOR}/span[starts-with(normalize-space(.), {{}})]/preceding-sibling::input"  # noqa
```

Comment:
`starts-with(normalize-space(.), {{}})` why are double curly braces needed here?

Reply:
Since it's an f-string but we call .format on it later